src/routes/auth/auth.controller.ts

```diff
@@ -0,0 +1,47 @@
+import { Body, Controller, Get, Post, Req, HttpCode } from '@nestjs/common';
+import { ApiExcludeController } from '@nestjs/swagger';
+import { Request } from 'express';
+import { ValidationPipe } from '@/validation/pipes/validation.pipe';
+import { AuthService } from '@/routes/auth/auth.service';
+import { VerifyAuthMessageDtoSchema } from '@/routes/auth/entities/schemas/verify-auth-message.dto.schema';
+import { VerifyAuthMessageDto } from '@/routes/auth/entities/schemas/verify-auth-message.dto';
+
+/**
+ * The AuthController is responsible for handling authentication:
+ *
+ * 1. Calling `/v1/auth/nonce` returns a unique nonce to be signed.
+ * 2. The client signs this nonce in a SiWe message, sending it and
```

Comment:
The spec mentions it as `SIWE`. We should keep it that way in my opinion.

Reply:
Changed in 52e1c66649c3a40858d05b1258748cfe621b3b26.